packages/e2e-tests/specs/editor/various/block-mover.test.js

```diff
@@ -18,6 +18,10 @@ describe( 'block mover', () => {
 		// Select a block so the block mover is rendered.
 		await page.focus( '.block-editor-block-list__block' );
 
+		// Move the mouse to show the block toolbar
+		await page.mouse.move( 0, 0 );
+		await page.mouse.move( 10, 10 );
```

Comment:
This is necessary because the typing observer does not set `isTyping` to `false` when a text field receives focus. Perhaps this can be adjusted in a follow-up PR.